tests/behaviour/util.rs

```diff
@@ -19,8 +19,158 @@
  * under the License.
  */
 
+use std::collections::HashMap;
+
+use chrono::{NaiveDateTime, NaiveTime};
 use cucumber::gherkin::Step;
+use futures::{
+    stream::{self, StreamExt},
+    TryFutureExt, TryStreamExt,
+};
+use regex::{Captures, Regex};
+use typedb_client::{
+    answer::ConceptMap,
+    concept::{Annotation, Attribute, Concept, Entity, Relation, Value},
+    transaction::concept::api::ThingAPI,
+    Result as TypeDBResult,
+};
+use typeql_lang::parse_query;
+
+use crate::behaviour::Context;
 
 pub fn iter_table(step: &Step) -> impl Iterator<Item = &str> {
     step.table().unwrap().rows.iter().flatten().map(String::as_str)
 }
+
+pub fn iter_map_table(step: &Step) -> impl Iterator<Item = HashMap<&String, &String>> {
```

Comment:
I'd suggest these should be called `iter_table` and `iter_table_map` to keep these two methods in sync.